ByteBuffer definition duplicates

chaoyli opened the pull request:
Change ByteBuffer to StorageByteBuffer in olap/byte_buffer.h.
Class definition of ByteBuffer duplicates between olap/byte_buffer.h and util/byte_buffer.h.
All of the two classes has a function names as remaining().
Some place which want to call remaining() of util/byte_buffer.h is linked to the other remaining() function of olap/byte_buffer.h

imay reviewed (changes requested):
change your commit subject

Commit:
Change ByteBuffer to StorageByteBuffer in olap/byte_buffer.h.
Class definition of ByteBuffer duplicates between olap/byte_buffer.h and util/byte_buffer.h.
All of the two classes has a function names as remaining().
Some place which want to call remaining() of util/byte_buffer.h is linked to the other remaining() function of olap/byte_buffer.h
